src/app/policies/routes.ts

```diff
@@ -40,38 +40,22 @@ export const routes = (store: Store<State>) => {
               path: '',
               name: `${prefix}`,
               redirect: (to) => {
-                let item = store.state.policyTypes.find((item) => {
-                  if (!(item.name in store.state.policyTypeTotals)) {
-                    return false
-                  }
-
-                  return store.state.policyTypeTotals[item.name].total !== 0
-                })
-
-                if (item === undefined) {
-                  item = store.state.policyTypes[0]
-                }
-
-                if (item === undefined) {
-                  return { name: 'home' }
-                }
-
```

Comment:
**Comment**: This was the logic that made us consider to just use a more or less arbitrary policy type as the default target as we would otherwise not be able to get rid of the global policy type state. We picked CircuitBreaker as most users will have a policy of this type via the default mesh policies.